pymeasure/instruments/eurotest/eurotestHPP120256.py

```diff
@@ -0,0 +1,417 @@
+    )
+
+    voltage_ramp = Instrument.control(
+        "STATUS,RAMP", "RAMP,%dV/s",
+        """ A integer property that represents the ramp speed (V/s) of the output voltage of the
+        HV Source. This property can be set.""",
+        # When this property acts as get, the instrument will return a string like this:
+        # "RAMP, RANGE=3000V/s, VALUE=1000V/s", then voltage_ramp will return 1000.0,
+        # hence the convenience of the get_process.
+        validator=strict_range,
+        values=VOLTAGE_RAMP_RANGE,
+        get_process=lambda r:
+        float(EurotestHPP120256.regex.search(r[2].strip()).groups()[0])
+    )
+
+    voltage = Instrument.measurement(
+        "STATUS,MU",
+        """ Measures the actual output voltage of the HV Source (kV).""",
+        # This property is a get so, the instrument will return a string like this:
+        # "U, RANGE=3.000kV, VALUE=2.458kV", then voltage will return 2458.0,
+        # hence the convenience of the get_process.
+        get_process=lambda r:
+        float(EurotestHPP120256.regex.search(r[2].strip()).groups()[0])
+    )
+
+    voltage_range = Instrument.measurement(
+        "STATUS,MU",
+        """ Returns the actual output voltage range of the HV Source (kV).""",
+        # This property is a get so, the instrument will return a string like this:
+        # "U, RANGE=3.000kV, VALUE=2.458kV", then voltage_range will return 3000.0,
+        # hence the convenience of the get_process.
+        get_process=lambda r:
+        float(EurotestHPP120256.regex.search(r[1]).groups()[0])
+    )
+
+    current = Instrument.measurement(
+        "STATUS,MI",
+        """ Measures the actual output current of the power supply (mA).""",
+        # This property is a get so, the instrument will return a string like this:
+        # "I, RANGE=5000mA, VALUE=1739mA", then current will return a 1739.0,
+        # hence the convenience of the get_process."""
+        get_process=lambda r:
+        float(EurotestHPP120256.regex.search(r[2].strip()).groups()[0])
+    )
+
+    current_range = Instrument.measurement(
+        "STATUS,MI",
+        """ Measures the actual output current range of the power supply (mA).""",
+        # This property is a get so, the instrument will return a string like this:
+        # "I, RANGE=5000mA, VALUE=1739mA, then current_range will return a 5000.0,
+        # hence the convenience of the get_process.
+        get_process=lambda r:
+        float(EurotestHPP120256.regex.search(r[1].strip()).groups()[0])
+    )
+
+    enable_kill = Instrument.setting(
+        "KILL,%s",
+        """ Enables or disables (True/False) the kill function of the HV source.
+         When Kill is enabled yellow led is flashing and the output
+         will be shut OFF permanently without ramp if Iout > IOUTmax.""",
+        validator=strict_discrete_set,
+        values={True: 'ENable', False: 'DISable'},
+        map_values=True
+    )
+
+    enable_output = Instrument.setting(
```

Comment:
```suggestion
    output_enabled = Instrument.setting(
```
follows a common pattern we try to establish. 
The instrument does not have a way to query that status?

Reply:
Yes. Now output_enabled and kill_enabled are Instrument.control. Thanks.